crates/autopilot/src/arguments.rs

```diff
@@ -379,32 +380,33 @@ pub enum FeePolicyKind {
     Volume { factor: f64 },
 }
 
-fn validate_factor(factor: f64) -> Result<(), String> {
-    if !(0.0..1.0).contains(&factor) {
-        return Err(format!("Factor must be in the range [0, 1), got {factor}",));
-    }
+fn validate_factor(factor: f64) -> anyhow::Result<()> {
+    anyhow::ensure!(
+        (0.0..1.0).contains(&factor),
+        "Factor must be in the range ..."
```

Comment:
Sorry, didn't mean to actually have the `...`. I was just too lazy to finish typing. 😅

Reply:
I was too fast :sweat: